.editorconfig

```diff
@@ -62,7 +62,7 @@ dotnet_style_object_initializer = true:warning
 dotnet_style_prefer_auto_properties = true:warning
 dotnet_style_prefer_compound_assignment = true:warning
 dotnet_style_prefer_conditional_expression_over_assignment = true:warning
-dotnet_style_prefer_conditional_expression_over_return = true:warning
+dotnet_style_prefer_conditional_expression_over_return = false
```

Comment:
Szerintem ez nem elég, az IDE0046-t kell a többi IDEx rule-hoz hasonlóan disable-lelni, lásd a commentet feljebb.

Reply:
Pedig úgy látszik, hogy elég mert egyrészt nekem a readmeben jelzett dotnet build parancs simán lefutott lokálban (beállítottam external toolnak), másrészt a CI se panaszkodott.

Reply:
WTF :D. Mit rontottál el, hogy neked működik? :D

Na mindegy, akkor OK, de ez elég fura, mert korábban számomra az derült ki, hogy ez by design nem így működik.